provider/azureProvider.js

```diff
@@ -102,27 +82,14 @@ return constants.providerName;
 return this.parsedBindings;
   }
 
-  getSetting (key) {
-    // Loop through environment variables looking for the keys, case insentivie
-    for (var k in process.env) {
-      if (process.env.hasOwnProperty(k)) {
-        if (k.toLowerCase() === key.toLowerCase()) {
-      return process.env[k];
-        }
-      }
-    }
-  }
+  Login() {
+    return login({ interactiveLoginHandler: (code) => {
+        this.serverless.cli.log(`Paste this code (copied to your clipboard) into the launched browser, and complete the authentication process: ${code}`);
```

Comment:
Does SP user created here show up in [portal](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal)? If yes, I think we should log indicating SP user being created.